Calculator/readme.md

```diff
@@ -0,0 +1,8 @@
+I have added a calculator web page. I have used simple HTML, CSS and JAVASCRIPT in it.
```

Comment:
Change this to contain information on the snippet from a third person point of view!

Reply:
Sir please check now.